palace/linalg/superlu.cpp

```diff
@@ -36,40 +36,40 @@ int GetNpDep(int np, bool use_3d)
 }  // namespace
 
 SuperLUSolver::SuperLUSolver(MPI_Comm comm, int sym_fact_type, bool use_3d, int print_lvl)
-  : mfem::SuperLUSolver(comm, GetNpDep(Mpi::Size(comm), use_3d))
+  : solver(comm, GetNpDep(Mpi::Size(comm), use_3d))
 {
   // Configure the solver.
   if (print_lvl > 1)
   {
-    if (npdep_ > 1)
+    if (GetNpDep(Mpi::Size(comm), use_3d) > 1)
```

Comment:
Why not `solver.npdep_`?

Reply:
I switched to the above originally when npdep was a protected variable, just hadn't updated it.